MACE_Frontend/src/components/aircraft-hud/single/index.tsx

```diff
@@ -211,10 +207,20 @@ export default React.memo((props: Props) => {
         setAltitude(parseFloat(value));
     };
 
-    const toggleRecordGoHere = () => {
-        setRecordGoHere(!recordGoHere);
+    const updateTarget = (field: string, value: number) => {
+        let newTarget = props.target;
+        if (field == "lat") { newTarget["lat"] = value; }
```

Comment:
In javascript, there is a difference between `==` and `===`. This does a good job of explaining: https://codeahoy.com/javascript/2019/10/12/==-vs-===-in-javascript/.

Before we merge this PR, please go through this file and verify that you mean `==` or `===`. My hunch is that you actually want `===` here.